gapii/cc/.gitignore

```diff
@@ -16,5 +16,5 @@
 /vulkan_VkPhysicalDevice_exports.cpp
 /vulkan_VkQueue_exports.cpp
 /*_spy.h
-/*_spy_[0-9].cpp
+/*_spy_[0-3].cpp
```

Comment:
I had a bunch'o'files knocking around that I kept on thinking were being generated and used. They weren't, they were left over from when you reduced the number of generated files.